packages/svelte/tests/runtime-runes/samples/event-attribute-template/main.svelte

```diff
@@ -1,10 +1,6 @@
 <script>
-	import { log } from './log.js';
-
-	function send() {
-		log.push("onclick")
-	}
+	let a = $state(0);
+	let b = $state(0);
 </script>
 
-{undefined}<hr/>
-<button onclick={send}>Send Event</button>
+{undefined}<button onclick={() => a += 1}>{a}</button>{undefined}<button onclick={() => b += 1}>{b}</button>
```

Comment:
Why did you change the test?

Reply:
because a) the indirectness (pushing to an array in a separate module) is weird, and hard to debug in a REPL compared to watching something change, and b) it only covers the case where the `{undefined}` is at the start of a fragment, rather than in the middle as well